process/sig-lifecycle.md

```diff
@@ -14,6 +14,9 @@ The SIG life cycle begins with interested contributors deciding to undertake thi
 
 ## To become `Active`:
 
```

Comment:
I am purposely omitting a recommendation of a group meeting before being considered a SIG